lib/foreman_remote_execution_core/script_runner.rb

```diff
@@ -8,6 +8,8 @@
 
 module ForemanRemoteExecutionCore
   class ScriptRunner < ForemanTasksCore::Runner::Base
+    attr_reader :execution_timeout_interval
```

Comment:
It seems we still need to override the `timeout_interval` method, to return the `@execution_timeout_interval` value